language/lexer/lexer.go

```diff
@@ -404,19 +404,13 @@ func blockStringValue(in string) string {
 	}
 
 	// Remove leading blank lines.
-	for {
-		if isBlank := lineIsBlank(lines[0]); !isBlank {
-			break
-		}
+	for len(lines) > 0 && lineIsBlank(lines[0]) {
```

Comment:
Consolidates the following couple of loops to remove blank lines, follows the graphql-js reference implementation:

```javascript
  // Remove leading and trailing blank lines.
  while (lines.length > 0 && isBlank(lines[0])) {
    lines.shift();
  }
  while (lines.length > 0 && isBlank(lines[lines.length - 1])) {
    lines.pop();
  }
```
- Ref: [Link](https://github.com/graphql/graphql-js/blob/master/src/language/blockString.js#L42-L48)